### DISCUSSION TITLE
Tutorial for Azure Web App

### DISCUSSION THREAD

**Discussion by wilterporto** [requarks/wiki#7187] [General]
I need a step-by-step installation tutorial on Azure. 

I was unable to connect the application to sgdb.

I followed the step-by-step instructions described at https://docs.requarks.io/install/azurewebapp, but I did not find the application configuration in Azure.

**Comment by maurtvedt** [comment]
I am trying to follow the same tutorial, also tried putting my db-configurations as Environment Variables (Which I assumed "Application configuration" could now be called) but these varaibles does not seem to be picked up by the container as it is unable to connect to the db, and the host it is reporting in log is not the one I have entered as environment variable, but just the name of the WebApp.
